### PR TITLE
Start sync job!

### PR DESCRIPTION
**Pull request by Lapis-Apple**

### CODE REVIEW

**Review by koumaza — APPROVED** [review]
Good job!
But I am not chinese-native, so I assign @justforlxz.

File: locale/zh-CN/How-to-Setup.md [hunk 1]
```diff
@@ -91,6 +92,11 @@ net stop lxssmanager && net start lxssmanager
 [root@PC-NAME]$ sudo pacman-key --populate
 ```
 
+### 安装修改版 GLibc (WSL1 环境下必需)
+Arch Linux 的官方 glibc 包是为 Linux 4.4 以上版本设计的，因此在 WSL1 中会完全无法使用。
+
+WSL1 用户 **必须** 跟随[这些](Known-issues.md#systemdsystemctl)步骤修改 GLibc 后才可使用。
```
Comment (yuk7) [inline locale/zh-CN/How-to-Setup.md:98]: Sorry, please link to Known-issues.md#wsl1--wsl2

File: locale/zh-CN/How-to-Setup.md [hunk 2]
```diff
@@ -36,6 +36,7 @@ has_children: true
 3. 双击安装 appx 文件。
 
 ## 完成安装后的操作
+### [若你使用 WSL1 ，你将**必须**修改一下 glibc 包。更多详情，请查看已知问题章节。](Known-issues.md#systemdsystemctl)
```
Comment (yuk7) [inline locale/zh-CN/How-to-Setup.md:39]: Sorry, please link to Known-issues.md#wsl1--wsl2

**Review by yuk7 — CHANGES_REQUESTED** [review]
Could you fix link?
Sorry for my mistakes...

**Comment by Lapis-Apple** [comment]
> Good job!
> But I am not chinese-native, so I assign @justforlxz.

Well...
I want to ask him for suggestion too, in fact, I've already sent PM for a translation problem.
But seems he's offline...

File: locale/zh-CN/How-to-Setup.md (hunk 2 above)
Comment (Lapis-Apple) [reply locale/zh-CN/How-to-Setup.md:39]: > Sorry, please link to Known-issues.md#wsl1--wsl2

Oh. Wait..

**Review by yuk7 — APPROVED** [review]
Thank you!

**Comment by Lapis-Apple** [comment]
Doing final check((

**Pull request merged**